src/SSHLibrary/pythonclient.py

```diff
@@ -101,15 +104,15 @@ def __init__(self, client, term_type, term_width, term_height):
         self._shell = client.invoke_shell(term_type, term_width, term_height)
 
     def read(self):
-        data = ''
+        data = bytes()
         while self._output_available():
             data += self._shell.recv(4096)
         return data
 
     def read_byte(self):
          if self._output_available():
             return self._shell.recv(1)
-         return ''
+         return bytes()
```

Comment:
I'd prefer `b''` over `bytes()`. Most importantly the usage should be consistent.